civcraftConfigs/tetconfig.yml

```diff
@@ -125,89 +125,97 @@ monster:
       spawn_chance: 0.05
       identifier: ocelot
       minimum_light_level: 7
-    JungleSkelly:
-     identifier: jungleskelly
-     type: SKELETON
-     range: 80
-     name: §kFive §2JungleSkelly §kFive 
-     spawn_chance: 0.2
-     drops:
-      PowBone:
-       material: BONE
-       amount: 2
-       lore: Coco Knocker
-       enchants:
-        KB1:
-         enchant: KNOCKBACK
-         level: 1
-     equipment:
-      BluntHotRod:
-       material: BONE
-       enchants:
-        KB8:
-         enchant: KNOCKBACK
-         level: 8
-        S2:
-         enchant: DAMAGE_ALL
-         level: 2
-      firechest451:
-       material: LEATHER_CHESTPLATE
-       enchants:
-        Prot451:
-         enchant: PROTECTION_FIRE
-         level: 451
-      firelegs451:
-       material: LEATHER_LEGGINGS
-       enchants:
-        Prot451:
-         enchant: PROTECTION_FIRE
-         level: 451
-      fireBoots154:
-       material: LEATHER_BOOTS
-       enchants:
-        FeatherFall4:
-         enchant: PROTECTION_FALL
-         level: 3
-#No glass helms permitted. Leave it to be whatever helmet wanted.
-     buffs:
-      CantDrown:
-       type: WATER_BREATHING
-       level: 2
-      Fast1:
-       type: SPEED
-       level: 1
-     on_hit_debuffs:
-      RunForestRun:
-       type: SPEED
-       level: 1
-       duration: 1s
-       chance: 1.0 
-      blinded:
-       type: BLINDNESS
-       level: 1
-       duration: 1s
-       chance: 1.0 
-      withered:
-       type: WITHER
-       level: 1
-       duration: 2s
-       chance: 1.0 
-     blocks_to_spawn_on:
-      - STONE
-      - DIRT
-      - GRASS
-      - STATIONARY_LAVA
-      - STATIONARY_WATER 
-      - LEAVES
-      - LEAVES_2
-     blocks_to_spawn_in:
-      - AIR
-      - STATIONARY_LAVA
-      - STATIONARY_WATER 
-     health: 160
-     despawn_on_chunk_unload: true
-     y_spawn_range: 16
-
+    jungleSkelly:
+      identifier: jungleskelly
+      type: SKELETON
+      range: 80
+      amount: 2
+      name: §kFive §2JungleSkelly §kFive 
+      spawn_chance: 0.08
+      drops:
+       tabbyboots:
+        material: CHAINMAIL_BOOTS
+        lore: "Tabbynya's Boots" 
```

Comment:
Probably shouldnt put player names there, unless you plan on making an item for every single player. Not sure whether you know, but you dont have to specify lore, enchants etc. You can just remove the option to default it.